packages/documentation/src/stories/misc/migration-guide/migrationv9-10.component.ts

```diff
@@ -107,24 +106,80 @@ export class MigrationV99Component extends LitElement {
 
       <section>
         <ol class="bubble-tea">
+          <li>
+            <h3>Introduction</h3>
+            <p>
+              Version 10 comes with a <span class="fw-bold">new look and cleaner codebase</span> —
+              and yes, a few breaking changes, all for good reason.
+            </p>
+            <p>
+              We’ve completely refreshed the design and reworked how components are built.
+              <span class="fw-bold">Bootstrap</span> and
+              <span class="fw-bold">Ng-Bootstrap</span> are gone; everything now runs on
+              <span class="fw-bold">Stencil Web Components</span>, which means the Design System
```

Comment:
The fact that they are built with Stencil should not matter to anyone, I think it could rather confuse in the worst case.

```suggestion
              <b>Bootstrap</b> and
              <b>Ng-Bootstrap</b> are gone; everything now runs on
              <b>Web Components</b>, which means the Design System
```